### DISCUSSION TITLE
Use of GPU Memory

### DISCUSSION THREAD

**Discussion by ghost** [SeisSol/SeisSol#735] [Q&A] [answered]
Could anyone tell me your experience running SeisSol-GPU about how many elements in your mesh and how many GPU memories occupied? Thanks a lot!

**Comment by ravil-mobile** [comment, marked as answer]
Hi @zsi2, based on my experience, 800000 elements with single precision and convergence order 6 can fit into 8GB of VRAM.  Consider to adjust `DEVICE_STACK_MEM_SIZE` env. variable (see https://seissol.readthedocs.io/en/latest/gpus.html) if your GPU has larger on-board memory and you would like to run bigger problems.

You may need to adjust LTS weight balancing (see https://seissol.readthedocs.io/en/latest/memory-requirements.html) if you are going to run your simulation in a multi-GPU setup
